### PR TITLE
Develop alexey

### PR DESCRIPTION
**Pull request by balexey88**
Updated docs
Updated tests

### CODE REVIEW

File: wp-stateless-elementor-addon.php [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 /**
  * Plugin Name: WP-Stateless - Elementor Addon
- * Plugin URI: https://udx.io
- * Description: Ensures compatibility with Elementor Page Builder. Sync css files generated by Elementor.
+ * Plugin URI: https://wp-stateless.github.io/
+ * Description: Ensures compatibility with Elementor Website Builder. Sync css files generated by Elementor.
  * Author: UDX
  * Version: 0.0.1
  * Text Domain: wpslea
```
Comment (planv) [inline wp-stateless-elementor-addon.php]: @balexey88 Please check it as well

File: readme.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Donate link: https://udx.io
+Tags: google, google cloud, google cloud storage, cdn, uploads, media, stateless, backup
+License: GPLv2 or later
+Requires PHP: 8.0
+Requires at least: 5.0
+Tested up to: 6.3.1
+Tested with Elementor: 3.16.5
+Stable tag: 0.0.1
+
+Provides compatibility between Elementor and WP-Stateless plugins.
```
Comment (planv) [inline readme.txt]: @balexey88 

`Elementor` -> `Elementor Website Builder`

**Review by planv — APPROVED** [review]

File: wp-stateless-elementor-addon.php (hunk 1 above)
Comment (balexey88) [reply wp-stateless-elementor-addon.php]: Changed text domain

File: readme.txt (hunk 2 above)
Comment (balexey88) [reply readme.txt]: Fixed, thanks!

**Review by planv — COMMENTED** [review]
@balexey88 I think we do not need to commit `vendor` folder

**Comment by balexey88** [comment]
> @balexey88 I think we do not need to commit `vendor` folder

It contains autoload for plugin classes. 
I agree that right now it can be removed, but in the future if the plugin will need new features we can require it.
In general I'm trying to keep the structure of all the plugins the same in order to simplify development and support.

We can also generate it as a part of plugin release, but in this case the result it can be different from testing environment.

**Pull request merged**